client/starwhale/core/dataset/store.py

```diff
@@ -388,6 +392,36 @@ def to_signed_http_backend(cls, dataset_uri: URI) -> ObjectStore:
             raise NoSupportError(f"{dataset_uri} is not dataset uri")
         return cls(backend=SWDSBackendType.SignedUrl, dataset_uri=dataset_uri)
 
+    @classmethod
+    def get_store(
+        cls, link: Link, owner: t.Optional[t.Union[str, URI]] = None
```

Comment:
The `owner` word has caused me some confusion, can it be replaced with another word?

Reply:
> The `owner` word has caused me some confusion, can it be replaced with another word?

It could be considered as the owner of the link. Now it is dataset only. But in the future, it could be evaulation/experiment and so on.